sdk/extensions/Microsoft.Azure.WebJobs.Extensions.Clients/tests/shared/AzureComponentFactoryWrapper.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+using Microsoft.Extensions.Azure;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Azure.WebJobs.Host.TestCommon
+{
+    public class AzureComponentFactoryWrapper : AzureComponentFactory
```

Comment:
The build is failing cause it can't find this class.

`AzureComponentFactory`

Is this building locally?

Reply:
Fixed the buikd